llvm/test/CodeGen/SPIRV/AtomicBuiltinsFloat.ll

```diff
@@ -1,4 +1,5 @@
 ; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
+; RUN: llc -O0 -mtriple=spirv64-amd-amdhsa %s -o - | FileCheck %s
```

Comment:
I'm not sure if there is a value in testing `spirv64-amd-amdhsa` triple in addition to `spirv64-unknown-unknown` for most of llvm/test/CodeGen/SPIRV tests. It makes sense to extend testing for vendor extensions mentioned in the description, but I suppose most of the tests cover core functionality which should work any `spir64-*` triple. Am I right?

Reply:
Yup, you are right; my thinking was to prevent anyone getting ideas and diverging the "flavoured" variant in ways that breaks its core functionality as a `spirv64` target, but that might be needlessly defensive. If we don't feel there's much value here, I'll revert the test spam around core functionality.

Reply:
I've rolled back most of these, except for a couple of places where there is a substantive difference (`spirv64-amdgcn-amdhsa` encodes the program address space in its DL, which yields slightly different IR in places).

Reply:
I am okay with either approach. I think we can assume that the core functionality will stay the same for all "flavors". In case of any changes in the future to this assumption, we could create larger tests covering multiple features, which will not provide any isolation value but just ensure the features are still there.